bin/cmd.js

```diff
@@ -5,8 +5,12 @@ var match = process.version.match(/v(\d+)\.(\d+)/)
 var major = parseInt(match[1], 10)
 var minor = parseInt(match[2], 10)
 
-if (major >= 11 || (major === 10 && minor >= 12)) {
-  require('standard-engine').cli(require('../options'))
+if (major >= 12 || (major === 12 && minor >= 20)) {
+  import('standard-engine').then(mod => {
+    import('../options.js').then(({ default: options }) => {
+      mod.cli(options)
+    })
+  })
```

Comment:
The purpose of this file is to run on really old versions of Node.js as well, that's why it's using e.g. `var` instead of `const`...

```suggestion
  import('standard-engine').then(function (engine) {
    import('../options.js').then(function (options) {
      engine.cli(options.default)
    })
  })
```

Reply:
Crap, `import` is a reserved word 😅 

<img width="415" alt="Screenshot 2021-09-14 at 10 37 41" src="https://user-images.githubusercontent.com/189580/133224826-815e431b-dee8-488e-a470-a6f5027918f9.png">

Use `eval`? 😄 

```suggestion
  eval('import("standard-engine")').then(function (engine) {
    eval('import("../options.js")').then(function (options) {
      engine.cli(options.default)
    })
  })
```

Reply:
Ha, eval/reserved word

> The purpose of this file is to run on really old versions of Node.js as well, that's why it's using e.g. var instead of const...

I figured it was

```js
eval('import("../options.js")')
```
won't this interfere with the cwd/path? feels like we would need a absolute path to options then...